lib/iris/tests/integration/format_interop/test_name_grib.py

```diff
@@ -72,21 +72,18 @@ def check_common(self, name_cube, grib_cube):
     def test_name2_field(self):
         filepath = tests.get_data_path(('NAME', 'NAMEII_field.txt'))
         name_cubes = iris.load(filepath)
-        # Check gribapi version, because we currently have a known load/save
-        # problem with gribapi 1v14 (at least).
-        gribapi_ver = gribapi.grib_get_api_version()
-        gribapi_fully_supported_version = \
-            (StrictVersion(gribapi.grib_get_api_version()) <
-             StrictVersion('1.13'))
+
+        # we currently have a known load/save
+        # problem with gribapi version and zero only data, where min == max.
+
```

Comment:
@marqh Is this still the most appropriate position for this comment to live?  It seems out of place now as it doesn't seem to refer to anything which is being tested here.  

I also don't really understand the comment now; which gribapi version has the problem? Is this just a placeholder for a test that will be added later when the load/save problem is addressed?